lists-ui/src/components/FiltersDrawer.tsx

```diff
@@ -0,0 +1,267 @@
+import { CSSProperties, Fragment, memo, useCallback, useState } from 'react';
+import { useDisclosure } from '@mantine/hooks';
+import {
+  Drawer,
+  ScrollArea,
+  Flex,
+  Title,
+  Stack,
+  Text,
+  Group,
+  UnstyledButton,
+  Paper,
+  Chip,
+  Collapse,
+  ActionIcon,
+} from '@mantine/core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faFilter,
+  faMinus,
+  faPlus,
+  faXmark,
+} from '@fortawesome/free-solid-svg-icons';
+import { FormattedMessage, FormattedNumber } from 'react-intl';
+import { FixedSizeList as List } from 'react-window';
+
+import classes from './FiltersDrawer.module.css';
+
+interface KV {
+  key: string;
+  value: string;
+}
+
+interface Facet {
+  key: string;
+  counts: { value: string; count: number }[];
+}
```

Comment:
Minor thing but these interfaces were already added to `types.ts` - so might want to import them?

Reply:
Good catch, forgot to remove the old declarations - fixed :)